spec/models/meursing_measure_spec.rb

```diff
@@ -0,0 +1,27 @@
+RSpec.describe MeursingMeasure do
+  it { expect(described_class.primary_key).to eq(:measure_sid) }
+
```

Comment:
`MeursingMeasure` is a filter onto an existing data set - it could do with some specs for the `where` clauses to check whats include/excluded?